xml/admin_caasp_advanced_config.xml

```diff
@@ -419,13 +420,13 @@ Node: node2.zbrbdl
     </para>
 <screen>
 # Set the number of PGs in the rbd pool to 512
-ceph osd pool set rbd pg_num 512
+c&prompt.cephuser;eph osd pool set rbd pg_num 512
 </screen>
    </sect3>
   </sect2>
 
   <sect2 xml:id="custom-cephconf-settings">
-   <title>Custom ceph.conf Settings</title>
+   <title>Creating Custom <filename>ceph.conf</filename> Settings</title>
```

Comment:
The previous sect2 title has no *ing ending, is it intentional or needs to be aligned?

Reply:
Intentional :) unless it is followed by a procedure, it should remain a noun, rather than a gerund - regardless of whether or not it is sect1, 2, or 3